src/app/searchsettings/searchsettings.component.spec.ts

```diff
@@ -19,4 +75,41 @@ describe('SearchsettingsComponent', () => {
     fixture.detectChanges();
   });
 
+
+  it('should create an instance', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should create a FooterNavbar Component', () => {
+    const footerNavbar = new FooterNavbarComponent();
```

Comment:
this is no way a test for instance if I import a results component and write
```
const resultscomponent = new ResultsComponent();
expect(resultscomponent).toBeTruthy();

```
this test gets passed.
even though searchsettings component doesn't have a results component in it.